providers/intercom/objectNames.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 // Supported object names can be found under schemas.json.
+// NOTE: tickets can be queried only by using non-empty `Since` parameter.
```

Comment:
What is the behavior of the tickets search API when we pass in no filters? Would that list all the tickets? If that is the case, I think that's better behavior and would be useful for builders to receive a backfill of all tickets, and then receive only updated tickets. 

i.e. when Since is not present, call `POST /tickets/search` with no request body, else call `POST /tickets/search WHERE updated_at > Since`

Reply:
This is a great hack!
![image](https://github.com/user-attachments/assets/53d7ca6d-21ac-4274-b513-80328dab60ec)
Other objects can be retrieved using GET endpoints. Tickets is the only exception that requires `Search` to see any.

This deep connector has many exceptions now.

Reply:
I have updated the docs removing the NOTE on tickets: https://github.com/amp-labs/docs/pull/78.

Also the full historic read works (temporary replaced "conversations" with "tickets", without Since param):
![image](https://github.com/user-attachments/assets/b911f8ad-6945-4292-8e4c-0ace9d0ae199)